styles/components/install-preparation-box.less

```diff
@@ -9,7 +9,7 @@
   background: #fff;
   background-color: #fff;
 
-  padding: 10px;
+  padding: 15px;
```

Comment:
Any reason for that change? On my browser, it results in the command overflowing its line.
![screenshot from 2018-05-21 01-13-45](https://user-images.githubusercontent.com/908424/40284784-55e1a2c8-5c94-11e8-9fed-b3b5a387773e.png)

Reply:
oh, lets revert... 
i thought it would look good to have some margin from the code box to div.